<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="launch_prefix" default="" />
    <arg name="map_name"       default="compslam" />
        

    <!-- Launch pole_detection node -->
    <node name="pointcloud_stitcher_node" pkg="smb_slam" type="pointcloud_stitcher_node"
        output="screen" launch-prefix="$(arg launch_prefix)">
        <param name="input_cloud_topic" value="/velodyne_cloud_registered"/>
        <param name="accumulate_on_startup" value="true"/>
        <param name="output_path" value="$(find smb_slam)/maps/$(arg map_name).pcd"/>
    </node>
    

</launch>
